ui/app/AppLayouts/Chat/ChatColumn/ChatInput.qml

```diff
@@ -276,18 +289,52 @@ Rectangle {
         }
     }
 
+    Row {
+        id: messageLengthLimitContainer
+        width: 174
+        anchors.right: parent.right
+        anchors.rightMargin: Style.current.padding
+        anchors.verticalCenter: parent.verticalCenter
+        leftPadding: Style.current.halfPadding
+        anchors.verticalCenterOffset: -18
+
+        StyledText {
+            id: messageLengthLimit
+            text: (messageLimit - txtData.length).toString()
+            color: (messageLimit - txtData.length >= 0) ? Style.current.success : Style.current.danger
+            rightPadding: Style.current.halfPadding
+        }
+    }
+
     ChatButtons {
         id: sendBtns
-        height: parent.height
+        width: 174
```

Comment:
I don't think we need that width. `ChatButtons` should calculate its own width correctly